### PR TITLE
repositories: remove raise-dev-library.

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
The repository was archived.

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @MikeMcQuaid. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

**Review by MatteoPologruto — APPROVED** [review]
Done:
```
2023/07/05 15:00:44 Loaded 5962 libraries from DB
Removing RaiseDev
Success!
```

**Pull request merged**
